utils/validate_consistency.py

```diff
@@ -166,7 +166,7 @@ def validate_verdi_documentation():
     replace_block_in_file(filepath_verdi_overview, overview_block_start_marker, overview_block_end_marker, block)
 
     # Replacing the block with the commands of `verdi`
-    filepath_verdi_commands = os.path.join(ROOT_DIR, 'docs', 'source', 'verdi', 'verdi_user_guide.rst')
+    filepath_verdi_commands = os.path.join(ROOT_DIR, 'docs', 'source', 'reference', 'command_line.rst')
```

Comment:
@ramirezfranciscof I am attaching this comment to this line because I cannot do it to the one I would ideally do it, since that line has not be touched, but, could you please update line 181 in this file. It now says:
```
header_label = '.. _verdi_{name:}:'.format(name=name)
```
but we should update it to the new link label standards.
It should become
```
header_label = '.. _reference:command-line:verdi-{name:}:'.format(name=name)
```